lib/manager/components/version/FeedVersionDetails.js

```diff
@@ -51,12 +52,30 @@ export default class FeedVersionDetails extends Component<Props> {
       !!(errorCounts.find(ec => BLOCKING_ERROR_TYPES.indexOf(ec.type) !== -1))
   }
 
+  _mergeItemFormatter = (v: FeedVersion, i: number, activeVersion: FeedVersion) => {
+    const name = v.id === activeVersion.id
+      ? '(Cannot merge with self)'
+      : v.retrievalMethod === 'SERVICE_PERIOD_MERGE'
+        ? '(Cannot re-merge feed)'
+        : v.name
+    const disabled = v.retrievalMethod === 'SERVICE_PERIOD_MERGE' ||
+      v.id === activeVersion.id
```

Comment:
Nitpick: might be better to use if/else if blocks here to make things more readable and avoid duplicate code.

Reply:
Something like:
```
let name = v.name
    let disabled = false
    if (v.id === activeVersion.id) {
      name = '(Cannot merge with self)'
      disabled = true
    }
    if (v.retrievalMethod === 'SERVICE_PERIOD_MERGE') {
      name = '(Cannot re-merge feed)'
      disabled = true
    }
```
?

Reply:
Actually does it make sense at all to display versions that cannot be merged?

Reply:
I think it works either way. I'm inclined to just disable them so they don't appear to be missing to the user without explanation.